autoscale/src/main/java/com/sequenceiq/periscope/service/ScalingActivityService.java

```diff
@@ -66,10 +67,28 @@ public List<Long> findAllIdsOlderThanDurationForCluster(Cluster cluster, long du
                 .stream().map(ScalingActivity::getId).collect(Collectors.toList());
     }
 
+    public List<ScalingActivity> findAllInGivenIntervalForCluster(Cluster cluster, long durationInMinutes, Pageable pageable) {
```

Comment:
Method name is incorrect - It isn't between an interval

Reply:
Done